scenario17/jmeter/SecureSPA_OauthSecuredProxy.jmx

```diff
@@ -1349,7 +1349,7 @@
             <connectTime>true</connectTime>
           </value>
         </objProp>
-        <stringProp name="filename">01-SecureSPA_OauthSecuredProxy.jtl</stringProp>
+        <stringProp name="filename">/home/shavantha/Desktop/gitstuff/identity-test-integration/scenario17/01-SecureSPA_OauthSecuredProxy.jtl</stringProp>
```

Comment:
Remove the local file path location and include the run-scenario.sh to execute the scenario